modules/openapi-generator/src/main/resources/swift4/api.mustache

```diff
@@ -54,7 +55,7 @@ open class {{classname}} {
         {{/returnType}}
         }
     }
-
+{{/useRxSwift}}{{/usePromiseKit}}
```

Comment:
@d-date minor suggestion: using the following instead to skip blank lines or trailing spaces in the output:
```
    {{^usePromiseKit}}
    {{^useRxSwift}}
....
    {{/useRxSwift}}
    {{/usePromiseKit}}
```

Reply:
@wing328  Thank you! Fix it.